lib/main-menu.js

```diff
@@ -153,6 +153,19 @@ var view = {
   label: 'View',
   submenu: [
     {
+      label: 'Toggle Full Screen',
+      accelerator: (() => {
+        if (process.platform === 'darwin') return 'Ctrl+Command+F'
+        return 'F11'
+      })(),
+      click: (item, focusedWindow) => {
```

Comment:
It's one the optional variables defined when calling the `click` function and it corresponds to the MenuItem. You can get a better idea about it [here](http://electron.atom.io/docs/api/menu-item/#new-menuitemoptions), the official electron API docs :)